<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>898</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>651</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_main">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QLabel" name="playerAvatar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>avatar</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerName">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerId">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="rankIcon">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>icon</string>
      </property>
     </widget>
     <widget class="QLabel" name="rankStar">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>star</string>
      </property>
     </widget>
     <widget class="QLabel" name="wlZn">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wl</string>
      </property>
     </widget>
     <widget class="QLabel" name="wl">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wl</string>
      </property>
     </widget>
     <widget class="QLabel" name="wrZn">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wr</string>
      </property>
     </widget>
     <widget class="QLabel" name="wr">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wr</string>
      </property>
     </widget>
     <widget class="QLabel" name="mph">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MOST PLAYED HEROES</string>
      </property>
     </widget>
     <widget class="QLabel" name="lm">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>350</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LATEST MATCHES</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="lastMatch">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>441</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="f">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>90</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>FRIENDS</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="friends">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>130</y>
        <width>331</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="mostHero">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>441</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="updateStat">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Update stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_player">
     <attribute name="title">
      <string>Search player by id</string>
     </attribute>
     <widget class="QLabel" name="mph2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MOST PLAYED HEROES</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerAvatar2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>avatar</string>
      </property>
     </widget>
     <widget class="QLabel" name="lm2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>350</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>LATEST MATCHES</string>
      </property>
     </widget>
     <widget class="QLabel" name="rankIcon2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>icon</string>
      </property>
     </widget>
     <widget class="QLabel" name="f2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>90</y>
        <width>351</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>FRIENDS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="updateStat2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Update stat</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="mostHero2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>441</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="changeId2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Search id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="changeIdEdit2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="lastMatch2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>390</y>
        <width>441</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="playerId2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLabel" name="wl2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wl</string>
      </property>
     </widget>
     <widget class="QLabel" name="playerName2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLabel" name="wr2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wr</string>
      </property>
     </widget>
     <widget class="QLabel" name="rankStar2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>20</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>star</string>
      </property>
     </widget>
     <widget class="QLabel" name="wrZn2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wr</string>
      </property>
     </widget>
     <widget class="QLabel" name="wlZn2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>20</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>wl</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="friends2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>130</y>
        <width>331</width>
        <height>221</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="settings">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>0</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
